### PR TITLE
Display show banner and Fanart background

### PR DESCRIPTION
**Pull request by p0psicles**
TODO:

- [ ] Add to snatch selection
- [ ] Disable transparency for result/episode table - who agree?

Cherry picked UI changes from SR.

@wimpyrbx Thank you for your hard work. We've added one commit, but that was because of some small merge conflicts.

### CODE REVIEW

**Comment by p0psicles** [comment]
Need to carefully check I haven't merged too much.
@OmgImAlexis can you also check?

**Comment by fernandog** [comment]
@p0psicles working for me

**Comment by duramato** [comment]
Manual search page looks funky, has same transparency as show's page but no background or banner, show's page its a bit slow. fanart.tv url links correctly but shows a diferent one on hover.
The items on the show's page are too transparent imo, or too dark, 
should be made brighter or the text bolder or both

File: .build/bower.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "SickRage",
+  "name": "sickrage",
```
Comment (OmgImAlexis) [inline .build/bower.json]: This should be "Medusa".

File: .build/package.json [hunk 2]
```diff
@@ -25,6 +25,8 @@
     "grunt-contrib-cssmin": "^0.14.0",
     "grunt-contrib-jshint": "^1.0.0",
     "grunt-contrib-sass": "^0.9.2",
+    "grunt-contrib-watch": "^0.6.1",
+    "grunt-contrib-qunit": "^0.7.0",
```
Comment (OmgImAlexis) [inline .build/package.json]: I'm pretty sure we don't use qunit and neither does Sickrage/Sickrage so why was this added?

**Comment by OmgImAlexis** [comment]
All of the CSS should be added using a new class. If the option is enabled then we should apply the class to the elements that need it.
The reason for this is a LOT of the CSS we use is just  a hacked together mess from before ech and we need to start cleaning it up for when Bootstrap 4 comes out.

**Comment by p0psicles** [comment]
Who's going to do the cleanup/fixup of this PR?

**Comment by fernandog** [comment]
its changing snatchSelection page transparency. I guess it shouldn't right? or fantart should be added to snatchSelection

**Comment by OmgImAlexis** [comment]
@fernandog that was one of the issues I was worried about since they're purposely changing the layout to add banners, etc. behind the content.

**Comment by labrys** [comment]
@fernandog Personally I think fanart should be added to snatch select.

**Comment by fernandog** [comment]
@wimpyrbx can you fix the 'TODO' in first post?

**Comment by labrys** [comment]
@wimpyrbx I'd like to have this in v0.1.1.  If you would like to complete this and extend it to cover the snatch selection, let me know.  If not let me know and we can get someone else on it.  I may have time this weekend to hit it.

**Comment by p0psicles** [comment]
Can someone rebase this for @wimpyrbx

**Comment by fernandog** [comment]
done

**Comment by wimpyrbx** [comment]
sorry for being a noob here. what does this mean? do i need to do something or is my PR here ok?

**Comment by p0psicles** [comment]
It means that all the commits done in this PR, have been "replayed" on top of the latest develop commit. Meaning that you have all the changes of develop also in here now.

**Comment by Thraxis** [comment]
Rebased code and changed to use custom CSS when fanart is enabled.

Dark Theme:

image


Light Theme

image

File: sickbeard/server/web/config/general.py [hunk 3]
```diff
@@ -167,6 +167,8 @@ def saveGeneral(self, log_dir=None, log_nr=5, log_size=1, web_port=None, notify_
         sickbeard.HANDLE_REVERSE_PROXY = config.checkbox_to_value(handle_reverse_proxy)
 
         sickbeard.THEME_NAME = theme_name
+        sickbeard.FANART_BACKGROUND = fanart_background
+		sickbeard.FANART_BACKGROUND_OPACITY = fanart_background_opacity
```
Comment (p0psicles) [inline sickbeard/server/web/config/general.py]: Some how some tabs got into here, should be 4 spaces.

**Comment by Thraxis** [comment]
@p0psicles  *coughs*  I know you mentioned adding in a future PR... so I've not merged this in yet:

image

**Comment by fernandog** [comment]
@Thraxis is it possible to make tables not transparent?

**Comment by p0psicles** [comment]
@fernandog yes that's possible. But as transparency is applied to a top level element, all elements below that will also be effected. Only solution here, is to move the tables out of the parent elements. Which will be some work.

**Comment by fernandog** [comment]
@p0psicles ok. don't think its worth then. Unless all others agree table shouldn't be transparent.

**Comment by fernandog** [comment]
I also doesn't like that banner in the top right position. Can we remove that?

**Comment by p0psicles** [comment]
I think the banner in the top right is pretty cool actually.

**Comment by duramato** [comment]
I like the transparency, allows you you see the background without compromising the tables.  I also like the banner, I see it as another show's art just like the poster.

**Comment by fernandog** [comment]
@Thraxis still need to add the transparency/etc to snatchselection.mako

**Comment by p0psicles** [comment]
@fernandog I told him not do add that for now. I want to feature freeze it. And get it merged asap.

**Comment by fernandog** [comment]
Approved

**Comment by fernandog** [comment]
@Thraxis still in progress?

**Comment by Thraxis** [comment]
No.. Was just making sure it was rebased to the current dev and that no merge conflicts crept in

**Comment by fernandog** [comment]
@Thraxis nice!! thank you!

https://cloud.githubusercontent.com/assets/1131905/17071560/1e6239d8-5017-11e6-9033-a76e97859979.png
